### PR TITLE
✨(ashley) create custom settings module

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

In Ashley we want to redefine settings module if needed. We just created
a settings module importing every settings and the good one will be used
thanks to the DJANGO_CONFIGURATION environment variable

## Proposal

- [x] create custom settings module
- [x] bump release to 6.6.0

### CODE REVIEW

File: apps/ashley/templates/services/app/_env.yml.j2 [hunk 1]
```diff
@@ -14,3 +14,4 @@ DJANGO_SETTINGS_MODULE: "{{ ashley_django_settings_module }}"
 ELASTICSEARCH_HOST: "{{ ashley_elasticsearch_host }}"
 ELASTICSEARCH_INDEX_NAME: "{{ ashley_elasticsearch_index }}"
 ELASTICSEARCH_PORT: "{{ ashley_elasticsearch_port }}"
+PYTHONPATH: "/app"
```
Comment (sampaccoud) [inline apps/ashley/templates/services/app/_env.yml.j2:17]: is it related to arnold or better done in the ashley dockerfile?

File: apps/ashley/templates/services/app/configs/settings.py.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+from sandbox.settings import *
```
Comment (sampaccoud) [inline apps/ashley/templates/services/app/configs/settings.py.j2]: missing line break?

File: apps/ashley/templates/services/app/_env.yml.j2 (hunk 1 above)
Comment (lunika) [reply apps/ashley/templates/services/app/_env.yml.j2:17]: This is only related to arnold

File: apps/ashley/templates/services/app/secret.yml.j2 [hunk 3]
```diff
@@ -15,3 +15,8 @@ data:
   DJANGO_AWS_SECRET_ACCESS_KEY: "{{ ASHLEY_VAULT.DJANGO_AWS_SECRET_ACCESS_KEY | default('secret') | b64encode }}"
   DB_USER: "{{ ASHLEY_VAULT.DB_USER | default('edxapp_user') | b64encode }}"
   DB_PASSWORD: "{{ ASHLEY_VAULT.DB_PASSWORD | default('password') | b64encode }}"
+{% if ASHLEY_VAULT.SECRETS is defined %}
+{% for k, v in ASHLEY_VAULT.SECRETS.items() %}
+  {{ k }}: {{ v | default('') | b64encode }}
+{% endfor %}
+{% endif %}
```
Comment (jmaupetit) [inline apps/ashley/templates/services/app/secret.yml.j2]: A carriage return is missing here.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**